tests/test_adapter.py

```diff
@@ -324,3 +325,26 @@ def forward_pre_hook(module, input):
 
         self.assertEqual((1, 128, model.config.decoder.vocab_size), out[0].shape)
         self.assertEqual(2, calls)
+
+
+@require_torch
+class T5AdapterTest(
```

Comment:
Since T5 supports flex heads & conversion `ModelClassConversionTestMixin` should be implemented (see `GPT2ClassConversionTest` for an example).